resources/keeper.yaml

```diff
@@ -186,3 +196,13 @@ spec:
         - name: cluster-default-ssl
           secret:
             secretName: cluster-default-ssl
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: telegraf-influxdb-creds
+  namespace: default
+data:
+  username: dGVsZWdyYWY=
+  password: dGVsZWdyYWY=
```

Comment:
For sure, but should be `c3VwZXJTZWN1cmVQYXNzd29yZAo=`.
Your password after decoding will be with trailing newline and will not work normally. Always remember about trailing newlines. For, example, if u create base 64 encoded password with echo, use `echo -n "password" | base64`. :smiley: